backend/db/config.py

```diff
@@ -76,6 +76,7 @@ def get_pg_connect_url(local=False):
 if CONFIG["server"] == 'mysql':
     BRAND_NEW_DB_URL = BRAND_NEW_DB_URL + '?charset=utf8mb4'
 DB_URL = BRAND_NEW_DB_URL.replace(f'{CONFIG["port"]}', f'{CONFIG["port"]}/{CONFIG["db"]}')
+REFRESH_JOB_MAX_HRS = 6  # see is_refresh_active() for documentation on this var
```

Comment:
## Strange: Multiple long running refreshes
<img width="1333" alt="multiple long running refresh" src="https://github.com/jhu-bids/TermHub/assets/13045020/d0c7dcbb-3984-43e9-8dcc-1ae8b2d9dc82">

This is related but this PR does not address this issue. I just want to raise the fact that I'm not sure why this is happening. There should only ever be 2 refresh actions going simultaneously, and the newer one should only be running for a short time. It should only get to the point at the beginning of the refresh where it sees another one is active and should quit out immediately. However in this screenshot, 2 of the refreshes were going for like an hour. I have now idea how this could happen, even after checking the logs. The logs were roughly identical, and both were doing the up-to-2 hour wait for the derived tables refresh to complete.